src/smolvm/cli/main.py

```
@@ -1876,14 +1876,22 @@ def _vmm_for_host() -> Vmm:
 
 
 def _published_path_enabled() -> bool:
-    """Opt-in switch for the published-image launch path.
+    """Opt-out switch for the published-image launch path.
 
-    Set ``SMOLVM_USE_PUBLISHED=1`` to bypass the install-at-boot flow and
-    download a pre-built rootfs from GitHub Releases instead. Default is
-    off — flipped on once the published flow has been smoke-tested
-    end-to-end (separate PR).
+    Default ON (post-0.0.14a0): ``smolvm <preset> start`` pulls a pre-built
+    image from GitHub Releases for fast first-boot (~5–10 s vs 60–120 s for
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)